h2o-algos/src/test/java/hex/tree/drf/DRFTest.java

```diff
@@ -1949,6 +1950,91 @@ private void checkDeepLeafNodeAssignmentConsistency(int depth, int extraObservat
     }
   }
 
+  @Test
+  public void testCategoricalSplitNAvsREST() {
+    Scope.enter();
+    try {
+      String[] dataCol1 = ArrayUtils.toString(ArrayUtils.seq(0, 64));
+      String[] dataCol2 = new String[dataCol1.length];
+      double[] response = new double[dataCol1.length];
+      double[] weights = new double[response.length];
+      for (int i = 0; i < response.length; i++) {
+          if (i <= 50) {
+              response[i] = i / 100.0;
+              weights[i] = 1.0;
+              dataCol2[i] = null;
+          } else if (i <= 57) {
+              response[i] = 10;
+              weights[i] = 100;
+              dataCol2[i] = null;
+          } else {
+              response[i] = 11;
+              weights[i] = 100;
+              dataCol2[i] = i % 2 == 0 ? "A" : "B";
+          }
+      }
+      
+      Frame f = new TestFrameBuilder()
+              .withColNames("C0", "C1", "response", "weight")
+              .withVecTypes(Vec.T_CAT, Vec.T_CAT, Vec.T_NUM, Vec.T_NUM)
+              .withDataForCol(0, dataCol2)
+              .withDataForCol(1, dataCol1)
+              .withDataForCol(2, response)
+              .withDataForCol(3, weights)
+              .build();
+      f.vec(0).setNA(0);
+
+      System.out.println(f.toTwoDimTable(0, (int) f.numRows(), false));
```

Comment:
I don't know if we are ok with souths in tests, I would rather use LOG.info

Reply:
LOG.info will make it log 2x - once in H2O logs and the second time in stdout. Since we only collect the stdout and the actual logs will roll off anyway - isn't sysout better?